addons/l10n_fr_pos_cert/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.l10n_fr_pos_cert</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="point_of_sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <form>
+                <field name="country_code" invisible="1"/>
+            </form>
+            <xpath expr="//div[@class='o_setting_left_pane mt-4']" position="attributes">
+                <attribute name="attrs">{'invisible': [('country_code', '==', 'FR')]}</attribute>
```

Comment:
this seems to work and be used in odoo, but I don't remember seeing `==` in a domain, and it is a little confusing since it would cause an error in python, so I would prefer `=`

```suggestion
                <attribute name="attrs">{'invisible': [('country_code', '=', 'FR')]}</attribute>
```